moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -151,9 +151,6 @@ export const ProjectActivityLogTableMaps = {
       milestone_name: {
         label: "name",
       },
```

Comment:
it should be okay to delete this entry because the formatter will ignore any undefined fields but lmk if we need to keep it!